src/components/PopoverMenu.tsx

```diff
@@ -336,7 +336,7 @@ PopoverMenu.displayName = 'PopoverMenu';
 export default React.memo(
     PopoverMenu,
     (prevProps, nextProps) =>
-        lodashIsEqual(prevProps.menuItems, nextProps.menuItems) &&
+        lodashIsEqual(prevProps.menuItems.length, nextProps.menuItems.length) &&
```

Comment:
Don't we need to compare the menu items and not just their lengths in order to show the error

Reply:
@s77rt Sorry, that was my mistake. I forgot to revert the change in `PopoverMenu` when running perf-test locally and mistakenly thought it was fixed. But the current fail test come from `AttachmentPickerWithMenuItems`